ivy/functional/frontends/tensorflow/nn.py

```diff
@@ -1,490 +1,499 @@
+                f"{input_depth} vs {filters_depth}"
+            )
+        feature_group_count = input_depth // filters_depth
+    return ivy.conv_general_dilated(
+        input,
+        filters,
+        strides,
+        padding,
+        dims=num_spatial_dims,
+        data_format=data_format,
+        dilations=dilations,
+        feature_group_count=feature_group_count,
+    )
+
+
+@with_unsupported_dtypes({"2.11.1 and below": ("complex",)}, "tensorflow")
+@to_ivy_arrays_and_back
+def embedding_lookup(params, ids, max_norm=None, name=None):
+    return ivy.embedding(params, ids, max_norm=max_norm)
+
+
+@with_unsupported_dtypes({"2.12.0 and below": ("complex",)}, "tensorflow")
+@to_ivy_arrays_and_back
+def relu(features, name=None):
+    return ivy.relu(features)
+
+
+@to_ivy_arrays_and_back
+def relu6(features, name=None):
+    return ivy.relu6(features)
+
+
+@with_unsupported_dtypes({"2.12.0 and below": ("float16",)}, "tensorflow")
+@to_ivy_arrays_and_back
+def softmax(logits, axis=None, name=None):
+    return ivy.softmax(logits, axis=axis)
+
+
+@with_unsupported_dtypes({"2.12.0 and below": "float16"}, "tensorflow")
+@to_ivy_arrays_and_back
+def leaky_relu(features, alpha=0.2, name=None):
+    return ivy.leaky_relu(features, alpha=alpha)
+
+
+@to_ivy_arrays_and_back
+def crelu(features, axis=-1, name=None):
+    c = ivy.concat([features, -features], axis=axis)
+    return ivy.relu(c)
+
+
+@to_ivy_arrays_and_back
+def avg_pool(input, ksize, strides, padding, data_format="NWC", name=None):
+    if len(ivy.shape(input)) == 3:
+        return ivy.avg_pool1d(input, ksize, strides, padding, data_format=data_format)
+    elif len(ivy.shape(input)) == 4:
+        return ivy.avg_pool2d(input, ksize, strides, padding, data_format=data_format)
+    return ivy.avg_pool3d(input, ksize, strides, padding, data_format=data_format)
+
+
+# avg_pool3d
+@to_ivy_arrays_and_back
+def avg_pool3d(input, ksize, strides, padding, data_format="NDHWC", name=None):
+    return ivy.avg_pool3d(input, ksize, strides, padding, data_format=data_format)
+
+
+# avg_pool1d
+@to_ivy_arrays_and_back
+def avg_pool1d(input, ksize, strides, padding, data_format="NWC", name=None):
+    return ivy.avg_pool1d(input, ksize, strides, padding, data_format=data_format)
+
+
+# pool
+@to_ivy_arrays_and_back
+def pool(
+    input,
+    window_shape,
+    pooling_type,
+    strides=None,
+    padding="VALID",
+    data_format=None,
+    dilations=None,
+    name=None,
+):
+    return ivy.pool(
+        input,
+        window_shape,
+        pooling_type,
+        strides=strides,
+        padding=padding,
+        data_format=data_format,
+        dilations=dilations,
+    )
+
+
+# log_poisson_loss
+@to_ivy_arrays_and_back
+def log_poisson_loss(targets, log_input, compute_full_loss=False, name=None):
+    return ivy.log_poisson_loss(targets, log_input, compute_full_loss, name)
+
+
+# ctc_unique_labels
+def ctc_unique_labels(labels, out_idx=ivy.int32, name=None):

```

Comment:
Should add `@to_ivy_arrays_and_back` decorator on this. Also `ctc_unique_labels` doesn't take `out_idx` as an argument. 
You can check it out [here](https://www.tensorflow.org/api_docs/python/tf/nn/ctc_unique_labels)

Reply:
Hey I fixed it from your request, and I create a new PR because for some reason github detecting change and I cannot find the reason for it.
You can find my new PR in here:
https://github.com/unifyai/ivy/pull/17859

Sorry for my mistake